lib/rubocop/cop/rspec/factory_bot/dynamic_attribute_defined_statically.rb

```diff
@@ -31,13 +31,13 @@ class DynamicAttributeDefinedStatically < Cop
             (block (send nil? {:factory :trait} ...) _ { (begin $...) $(send ...) } )
           PATTERN
 
-          def_node_matcher :callback_with_symbol_proc?, <<-PATTERN
-            (send nil? {:before :after} sym (block_pass sym))
+          def_node_matcher :callback_or_sequence_with_symbol_proc?, <<-PATTERN
+            (send nil? {:before :after :sequence} sym (block_pass sym))
```

Comment:
maybe we need to skip block_pass on any arguments, not only the ones listed here?

Reply:
That is exactly my point. Not sure if we should handle proc in a recursive manner, or can still use the recursive_literal_or_const helper

Reply:
I think we could handle proc only on the top-level only for this purpose (as we only need to handle arguments to the function), which would let us re-use `recursive_literal_or_const`

Maybe @bquorning has an opinion on that?

Reply:
So something like `proc?(foo) || recursive_literal_or_const?(foo)`? Sounds good.